### PR TITLE
[DO NOT MERGE] Change code formatting according to dfmt rules

### PR DESCRIPTION
**Pull request by gavin-norman-sociomantic**

### CODE REVIEW

File: src/dhtnode/connection/DhtClient.d [hunk 1]
```diff
@@ -83,28 +79,26 @@ public class DhtNodeRegistry : Swarm.DhtNodeRegistry
 
     ***************************************************************************/
 
-    public this ( EpollSelectDispatcher epoll, ClientSettings settings,
-        IRequestOverflow request_overflow,
-        INodeConnectionPoolErrorReporter error_reporter )
+    public this (EpollSelectDispatcher epoll, ClientSettings settings,
+            IRequestOverflow request_overflow,
+            INodeConnectionPoolErrorReporter error_reporter)
```
[the remark this answers is not in the document]
Comment (stefan-koch-sociomantic) [reply src/dhtnode/connection/DhtClient.d]: I don't understand it either. maybe because of the parens ?

**Comment by gavin-norman-sociomantic** [comment]
See what I mean, though? There are a few oddities, but nothing that feels like a blocker to me.

**Comment by stefan-koch-sociomantic** [comment]
Currently the double-indent is consistent, but if it will be still there in future versions is dependent on whether it was intentional or not.
and I am not sure.

**Comment by matthias-wende-sociomantic** [comment]
This hasn't been touched for a year.

**Pull request closed without merging**